docs/Tools_Equipment/Wood_Working/Pillar_Drill.md

```diff
@@ -0,0 +1,19 @@
+# Pillar Drill
+
+![](beltsander.jpeg "beltsander.jpeg")
```

Comment:
Just to let you know this contains the old beltsander image

Reply:
There seem to be other sander-related things included - pillar drills don't need abrasive belts